minegauler/shared/utils.py

```diff
@@ -112,10 +112,7 @@ def __str__(self, mapping=None, cell_size=None):
             The size to display a grid cell as. Defaults to the maximum size of
             the representation of all the objects contained in the grid.
         """
-        # @@@LG Some attention please :)
-
-        # Use max length of object representation if no cell size given.
-        if cell_size is None:
+         if cell_size is None:
```

Comment:
Revert - the comment is useful, and you've (accidentally?) added a bad indentation.